### ISSUE TITLE
Test for moving between chapters with a screen reader

### ISSUE THREAD

**Issue by ways2read** [daisy/epub-accessibility-tests#6]
With some reading systems it can be impossible for a screen reader user to move to the next chapter without having to resort to gymnastics with the table of contents.

We don't have a test in the non-visual reading chapter that checks tfor this functionality. This can be added to the next version of the test book.

In December 2017 we found that 5 reading systems would pass this test out of 14 looked at.

**Comment by GeorgeKerscher** [comment]
In Microsoft Edge, I cannot go to the next chapter (spine item) without going back to the TOC. Currently in Edge, the screen reader user (jaws 2018)  does not know what the hierarchy  is; the sited person can see the indents, but these levels are not voiced.

This reinforces the need to put in a test for moving easily to the next file in the spine.  While a work around is to go to the TOC and move to the next item, Edge does not enable this. I believe we also agreed in discussions that going to the TOC was too difficult for normal use.

**Comment by marisademeglio** [comment]
Technically, these are two different things: 1. next chapter 2. next spine item. While in practice, they are often the same thing, they do not have to be, according to the specification.

**Comment by avneeshsingh** [comment]
Next chapter is an user oriented term.
Here we are talking about moving to the next html file i.e. next spine item.
Most of the reading systems use browsers under the hood, and screen readers provide good navigation on the same html page. But screen readers cannot move to next html page, so a keyboard shortcut is required to move to the next content document or the spine item.

**Comment by GeorgeKerscher** [comment]
I believe this is resolved in the non-visual test book. There is a test to go to the next file in the spine.
